lib/data_distributor/cpr/person.rb

```diff
@@ -0,0 +1,83 @@
+module DataDistributor
+  module CPR
+    #This class represents a person as known by cpr
+    class Person
+      attr_reader :data, :cpr
+
+      # @param [Dataset]
+      def initialize(data, cpr:)
+        @data = data
+        @cpr = cpr
+      end
+
+      # Person's first name(s)
+      def first_name
```

Comment:
```suggestion
       # @return [String] first names separated by space
      def first_names
```

Jeg ved ikke lige om beskrivelsen passer, men det var bare det jeg lige forventede.